Rakefile

```
@@ -114,6 +116,31 @@ namespace :gem do
     end
   end
 
+  desc "document all gems"
+  task docs: [:girdocs, :rubydocs]
+
+  task :girdocs do
+    packages_rb = packages.map { |pkg|
+      File.join(pkg, "lib", pkg + ".rb")
+    }
+    ygir_gem = Gem::Specification.find_by_name('yard-gobject-introspection')
+    ygir_lib = File.expand_path('lib/yard-gobject-introspection.rb',
+                                ygir_gem.full_gem_path)
+    args = [ygir_lib, * packages_rb]
+    all_args = %w(bundle exec yard doc -o ./yard_docs/gir/ --load).concat(args)
+    sh Shellwords.join(all_args)
```

We don't need to use `Shellwords` because `sh` accepts variable size arguments:

```suggestion
    sh(*all_args)
```